### DISCUSSION TITLE
Tell people about Copilot in Codespaces

### DISCUSSION THREAD

**Discussion by jlivingstonsg** [community/community#19082] [Codespaces] [answered]
----
You need to tell people about Copilot in Codespaces,  
because that is amazing.

People need to know how great that is.

Regards from Sweden
--------------------------------------

**Comment by reybard** [comment, marked as answer]
We're glad you're enjoying it. 😄 We use copilot internally to the point where it's a default extension on most codespaces we launch. We love it, too! Agreed that the [docs on copilot](https://docs.github.com/en/codespaces/codespaces-reference/using-github-copilot-in-codespaces) may not be drawing enough eyes to the [extension in the marketplace](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) so maybe we can signal boost that a bit with social media.
